packages/shared/notionable-hooks/src/summary/use-trade-summary.tsx

```diff
@@ -731,5 +731,37 @@ export function useTradeSummary(state: VaultTradeState | TradeState) {
     summary.push(walletTotal);
   }
 
+  const isDebt = tradeType === 'BorrowFixed' || tradeType === 'BorrowVariable';
+
+  const prefixSymbol = isDebt ? '-' : '+';
+
+  const earningsRow = {
+    isTotalRow: true,
+    isEarningsRow: true,
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    label: tradeType && Earnings[tradeType] ? Earnings[tradeType] : '',
+    value: {
+      data: [
+        {
+          displayValue: `${prefixSymbol}${earnings?.toDisplayStringWithSymbol()}`,
+          isNegative: false,
+          showPositiveAsGreen: !isDebt,
+        },
+      ],
+    },
+  };
+
+  if (
```

Comment:
Why is this if statement so involved? Shouldn't it just check if the earnings is undefined or not? In fact, why not just put this whole code block you added inside an:

```
if (earnings) {
...
}
```

Reply:
It is involved because just earnings wasn't enough to prevent a undefined value from being shown. We never want to show the earnings row on any of these pages so this was an effective way of preventing that.

Reply:
Would this be better 

` if (earnings && earnings.isPositive()) {`